Sources/Sentry/SentryClient.m

```diff
@@ -490,6 +498,28 @@ - (void)setUserIdIfNoUserSet:(SentryEvent *)event
     }
 }
 
+- (BOOL)isOOM:(SentryEvent *)event isCrashEvent:(BOOL)isCrashEvent
+{
+    return isCrashEvent && event.exceptions != nil && event.exceptions.count == 1 &&
+        [event.exceptions[0].type isEqualToString:SentryOutOfMemoryExceptionType] &&
+        [event.exceptions[0].value isEqualToString:SentryOutOfMemoryExceptionValue];
```

Comment:
M: should we check the mechanism instead of type and value?

Reply:
Makes sense. Changed it.